db/db.js

```diff
@@ -0,0 +1,50 @@
+const mongoose = require('mongoose');
+
+const sampleData = require('./sampleCommentData');
+
+const Schema = mongoose.Schema; 
+
+mongoose.connect('mongodb://localhost/commentsListViewAndSubmission');
+
+const db = mongoose.connection;
+db.on('error', () => console.log('error connecting to mongo'));
+db.once('open', () => console.log('connected to mongo'));
+
+const repliesSchema = new Schema({
```

Comment:
this can also be done in the same step where you're instantiating your schema on line 5.

Reply:
(sorry to clarify, you can make do something like 
commentSchema = mongoose.Schema and replySchema = mongoose.Schema

Reply:
Yeah, I was following the mongoose guide on that one. I find it easier to not have to mongoose.Schema for every new schema.